detection-rules/brand_impersonation_bbb.yml

```
@@ -0,0 +1,57 @@
+name: "Brand impersonation: the Better Business Bureau (BBB)"
+description: "Detect impersonation of the Better Business Bureau (BBB) from unsolicited sender"
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  and 
+  // display name contains bbb
+  (
+    regex.icontains(strings.replace_confusables(sender.display_name),
+                    '(?:\bBBB\b|better business bureau)'
+    )
+    // or levenshtein distance similar to the full name
+    or strings.ilevenshtein(strings.replace_confusables(sender.display_name),
+                            'better business bureau'
+    ) <= 1
+  )
+  // and the sender is not from the bbb and passes auth
```

```suggestion
  // doesn't contain a link to the legit method of responding to bbb complaints
  and not any(body.links,
          .href_url.domain.domain == "respond.bbb.org"
          and .href_url.path == "/respond/"
          and .display_url.domain.domain == "respond.bbb.org"
          and .display_url.path == "/respond/"
  )
  // and the sender is not from the bbb and passes auth
```
samples that will no longer match - https://platform.sublime.security/messages/hunt?huntId=01955081-2700-7e8e-814c-fec2d63674e3